alacritty/src/renderer/mod.rs

```diff
@@ -1573,21 +1591,27 @@ impl Atlas {
         let offset_x = self.row_extent;
         let height = glyph.height as i32;
         let width = glyph.width as i32;
+        let colored;
 
         unsafe {
             gl::BindTexture(gl::TEXTURE_2D, self.id);
 
             // Load data into OpenGL
+            let (format, buf, _) = match &glyph.buf {
+                BitmapBuffer::RGB(buf) => (gl::RGB, buf, colored = false),
+                BitmapBuffer::RGBA(buf) => (gl::RGBA, buf, colored = true),
+            };
```

Comment:
This is an interesting idea, however it took me like three reads to understand what's actually going on here.

Could we just simplify this to not use tuples to assign things please?

Having small codeblocks like this should be completely sufficient:
```rust
{
    color = false;
    (gl::RGB, buf)
}
```